doc/api/crypto.md

```diff
@@ -668,6 +668,14 @@ provided, `otherPublicKey` is expected to be a [`Buffer`][], `TypedArray`, or
 If `outputEncoding` is given a string will be returned; otherwise a
 [`Buffer`][] is returned.
 
+*Note*:
```

Comment:
Nit: I'd remove `*Note*:` and let the paragraph stand on its own.

Reply:
Done!
Updating the pr soon.
Thanks!